src/main/java/uk/gov/companieshouse/confirmationstatementapi/configuration/InterceptorConfig.java

```diff
@@ -14,6 +14,6 @@
 
     @Override
     public void addInterceptors(InterceptorRegistry registry) {
-        registry.addInterceptor(transactionInterceptor);
+        registry.addInterceptor(transactionInterceptor).excludePathPatterns("/confirmation-statement/company/**");
```

Comment:
I would use `includePathPatterns("/transaction/**") instead of exclude.
As we will only ever want this to run on transaction prefixed routes

Reply:
I was worried you'd say that.

Reply:
If it's all the same i'd like to kepp this as is for now, I did try the include option but it was causing the transaction path to return a different response and I don't want to investiagate that here as this PR is getting too big already.

Reply:
Ok, if you raise a task at the top of the backlog.
Something like TechDebt: Api: Transaction Interceptor, use includePathPattern instead of ExcludePathPattern